### ISSUE TITLE
Deploy with Installer (and updater)

### ISSUE THREAD

**Issue by luttje** [luttje/Key2Joy#30]
The application is getting more and more files in each release. It's cumbersome for users to to download a zip, extract it to a single location, and open a specific executable [(as they are instructed)](https://github.com/luttje/Key2Joy/releases/tag/0.5.3).

## Installer
It would be nicer to have an installer perform the above installation procedure and (optionally) create a shortcut on the Desktop and in the Start Menu.

## Updater
If there's enough time it would also be nice to have the app be able to update itself. The update would simply download the newest release from GitHub and replace the app with it.

**Comment by luttje** [comment]
If I have the time I'll work on this within a few weeks after #29 is completed.

**Comment by abart27** [comment]
Think we can leverage msix for this, and just ship those in our releases?


Perhaps even bothering with the MS store (😥😓)...

**Comment by luttje** [comment]
Heya, I'd have to look into it more. I do not have experience with MSIX much, nor with the MS store.

All I know is that I always look for a non MS Store option to download software. I imagine I'm not alone, so I'm wondering if it's worth the hassle.
